src/components/Header/AuthereumButton.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import styled from 'react-emotion'
+import getWeb3 from '../../api/web3'
+import AuthereumLogo from '../../assets/authereum_shield.svg'
+import { GlobalConsumer } from '../../GlobalState'
+
+const Authereum = styled('div')`
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  overflow: hidden;
+  margin-left: -18.5px;
+  margin-bottom: -42.5px;
```

Comment:
Integers only for pixel values. They'll be rounded anyway so let's make it explicit